AsyncDisplayKit/Private/ASStackUnpositionedLayout.mm

```diff
@@ -23,15 +23,16 @@
                                   const CGFloat stackMin,
                                   const CGFloat stackMax,
                                   const CGFloat crossMin,
-                                  const CGFloat crossMax)
+                                  const CGFloat crossMax,
+                                  const CGSize size)
 {
   const ASStackLayoutAlignItems alignItems = alignment(child.alignSelf, style.alignItems);
   // stretched children will have a cross dimension of at least crossMin
   const CGFloat childCrossMin = alignItems == ASStackLayoutAlignItemsStretch ? crossMin : 0;
   const ASSizeRange childSizeRange = directionSizeRange(style.direction, stackMin, stackMax, childCrossMin, crossMax);
-  ASLayout *layout = [child measureWithSizeRange:childSizeRange];
+  ASLayout *layout = [child layoutThatFits:childSizeRange parentSize:size];
```

Comment:
`-layoutThatFits:parentSize: is declared in a nonnull block in `ASLayoutable.h`, so the method already has a proper nullability annotation. The problem is that you cannot predict what can happen in production.
As already said above we try to prevent crashes in production and assert in development